### PR TITLE
micro interactions

### PR DESCRIPTION
**Pull request by pigstake**
@mikehmorrissey @nrswolf @brookekao

### CODE REVIEW

File: client/dot-votr.html [hunk 1]
```diff
@@ -3,7 +3,7 @@
   <meta name="viewport" content="width=device-width">
   <script src="//use.typekit.net/xpm7idq.js"></script>
   <script>try{Typekit.load({ async: true });}catch(e){}</script>
-  <link href='http://fonts.googleapis.com/css?family=Muli:400,400italic,300italic,300' rel='stylesheet' type='text/css'>
+  <link href="//maxcdn.bootstrapcdn.com/font-awesome/4.2.0/css/font-awesome.min.css" rel="stylesheet" media="all">
```
Comment (mikehmorrissey) [inline client/dot-votr.html]: @pigstake I think there is a meteor font-awesome package

Comment (pigstake) [inline client/dot-votr.html]: :+1: sweet, thanks

Comment (pigstake) [inline client/dot-votr.html]: going to change in next PR

File: client/assets/app/stylesheets/main.scss [hunk 2]
```diff
@@ -10,6 +10,7 @@
 @import ".meteor/local/build/programs/server/assets/packages/meta_susy/susy"; // Susy
 @import "../../vendor/breakpoint-2.x.x/breakpoint.scss"; // breakpoint.css
 @import "exponents.scss"; // exponent math function
+@import "../../vendor/hoverCSS-iconSpin.scss"; // iconSpin effect from HoverCSS
```
Comment (nrswolf) [inline client/assets/app/stylesheets/main.scss]: @pigstake why do these packages fail at file naming conventions...it is a mystery

File: client/assets/app/stylesheets/main.scss [hunk 3]
```diff
@@ -10,6 +10,7 @@
...
 
 // Object and shameful styles
 @import "shame.scss";
```
Comment (nrswolf) [inline client/assets/app/stylesheets/main.scss]: @pigstake :+1:

**Comment by nrswolf** [comment]
@pigstake looks good to me

**Pull request merged**